ports/stm32/boards/NUCLEO_G474RE/stm32g4xx_hal_conf.h

```diff
@@ -0,0 +1,61 @@
+/**
+  ******************************************************************************
+  * @file    stm32g4xx_hal_conf.h
+  * @author  MCD Application Team
+  * @brief   HAL configuration template file.
+  *          This file should be copied to the application folder and renamed
+  *          to stm32g4xx_hal_conf.h.
+  ******************************************************************************
+  * @attention
+  *
+  * <h2><center>&copy; Copyright (c) 2019 STMicroelectronics.
+  * All rights reserved.</center></h2>
+  *
+  * This software component is licensed by ST under BSD 3-Clause license,
```

Comment:
In order to not over complicate licensing, please redo this file basing it off, eg, `boards/NUCLEO_H743ZI/stm32h7xx_hal_conf.h`

Reply:
Done.